apps/src/lib/ui/accounts/ManageLinkedAccounts.jsx

```diff
@@ -223,6 +224,10 @@ class OauthConnection extends React.Component {
       ? `/users/auth/${id}/disconnect`
       : `/users/auth/${credentialType}?action=connect`;
 
+    const buttonDisabled =
+      !!disconnectDisabledStatus ||
+      (isCodeOrgBrowser() && displayName === 'Google Account');
```

Comment:
Will this work for non-English users? Would `credentialType === OAuthProvider.google` work?

Reply:
Fixed it - thanks!